front-end/package.json

```diff
@@ -51,7 +51,8 @@
   },
   "dependencies": {
     "@babel/runtime-corejs3": "^7.15.4",
-    "cypress-real-events": "^1.7.0"
+    "cypress-real-events": "^1.7.0",
+    "p-debounce": "^4.0.0"
```

Comment:
`p-debounce` is already included in `front-end/packages/robustar/package.json`, no need to add it here again. Please also revert all `package-lock.json` changes if you are not changing the libraries.

Reply:
Fixed.